go.mod

```diff
@@ -1,21 +1,12 @@
 module github.com/tetratelabs/log
 
-go 1.15
+go 1.16
 
 require (
-	github.com/BurntSushi/toml v0.3.1 // indirect
-	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/natefinch/lumberjack v0.0.0-20170531160350-a96e63847dc3
-	github.com/pkg/errors v0.8.0 // indirect
-	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/spf13/cobra v0.0.3
-	github.com/spf13/pflag v1.0.3
-	github.com/stretchr/testify v1.2.2 // indirect
-	go.uber.org/atomic v1.3.2 // indirect
-	go.uber.org/multierr v1.1.0 // indirect
-	go.uber.org/zap v1.9.1
-	google.golang.org/grpc v1.15.0
-	gopkg.in/natefinch/lumberjack.v2 v2.0.0 // indirect
-	gopkg.in/yaml.v2 v2.2.1 // indirect
+	github.com/spf13/cobra v1.1.3
+	github.com/spf13/pflag v1.0.5
+	go.uber.org/multierr v1.6.0 // indirect
+	go.uber.org/zap v1.16.0
+	google.golang.org/grpc v1.36.0
+	gopkg.in/natefinch/lumberjack.v2 v2.0.0
```

Comment:
changing the lumberjack package, then doing `go get -u ./...; go mod tidy` make this file simpler